### PR TITLE
[examples][frameworks][static-build] Add Nift

### PR DESCRIPTION
**Pull request by n-ham**

### CODE REVIEW

File: packages/now-static-build/src/frameworks.ts [hunk 1]
```diff
@@ -514,6 +514,13 @@ const frameworkList: Framework[] = [
     buildCommand: 'bundle exec middleman build',
     getOutputDirName: async () => 'build',
   },
+  {
+    name: 'Nift',
+    slug: 'nift',
+    buildCommand: 'nift build-all',
```
Comment (styfle) [inline packages/now-static-build/src/frameworks.ts:520]: Where does the `nift` binary come from? Is it on https://www.npmjs.com or another package registry?

Comment (n-ham) [reply packages/now-static-build/src/frameworks.ts:520]: Binaries are available with the release assets on GitHub, see https://github.com/nifty-site-manager/nsm/releases 

Ed. for v2.3.8 the link is https://github.com/nifty-site-manager/nsm/releases/download/v2.3.8/Vercel.zip

Each release will have a Vercel.zip asset containing `nift` and `nsm` binaries (they are identical to each other, but some users may override the build command and use `nsm`). You can also compile it yourself using `make VERCEL=1`.

Might be worth chatting to Leo and Matt who know about this stuff as well.

Comment (styfle) [reply packages/now-static-build/src/frameworks.ts:520]: Thanks, I'll ask them 👍

**Comment by EndangeredMassa** [comment]
This doesn't appear to be maintained anymore. If that's not true, feel free to re-open this PR.

**Comment by n-ham** [comment]
Definitely wasn't doing development for a while, however am laying the groundwork to do a big v3.0 with ssr style builds etc. (possibly rebranded as mantis, based on the mantis shrimp). 

This PR was back from when Vercel was going to sponsor Nift but then never really heard back after putting the incremental builds server side in etc., then had a bit of a breakdown a few months later from some old piracy software which was riddled with CP popping up with still-active forums when it was meant to be shutdown years prior. The way I was treated at the time only made the situation far far worse, throwing my life into more disarray and significantly lengthening recovery. 

Locally people insisted I was delusional about old p2p piracy software being riddled with CP despite providing links to numerous articles indicating otherwise along with pointing out there was thousands of results on google news search on the topic, gah sorry just a frustrating reminder of trying to talk sense to people when your mental health has deteriorated for good reason who have the power to hold you against your will and have you held down and injected against your will till you pass out and make all sorts of logical assumptions while refusing to even check any evidence that ought to convince them their assumptions are wrong. Props to the UN for calling out taking people's body autonomy away to the point of drugging them against their will as inhumane. 

Never is an appropriate place to try and stick up for myself a bit, but that was unfortunately the most recent of a small string of breakdowns which started while being paid a few grand a year to do lecturing for a class with 600+ students per year down here in Tasmania. Had already told them I was unwilling to continue working at such an unlivable wage and they still hounded me in person about finishing the work for them as I was telling them my father had unexpectedly passed away. Quite appropriately told them where to stick it then and there but was already facing homelessness at the time from such a pitiful amount of income while trying to chase a career in academia and my mental health deteriorated (. That kind of thing severely limits what one is allowed to do career wise etc. and makes it very hard to get people to interact with you afterwards, leading to a very viscous cycle to escape from. 

There's a lot of hate locally for the university down here these days thankfully, but it's still the same types of folks down there running it with many of the same academics who have sat by silently while peeps have been treated like that (a recent news article said 1/6 staff there couldn't afford food properly at various times in the last 12 months, complete circus). They even had a forced compulsory vote about them spending millions upon millions moving from a government granted campus near the city into the city recently with 75% of people voting not to support it, unfortunately that's not going to stop the university and government from steam rolling ahead against public opinion.

**Comment by EndangeredMassa** [comment]
Sorry to hear about your troubles, @n-ham. That's rough.

**Comment by n-ham** [comment]
Cheers, definitely a rollercoaster. Peeps from vercel have been some of the
nicest, most understanding and actually inclusive from my experience at
various stages of sharing that, while not advertising those qualities
nearly as much as many other communities. Kudos to everyone at vercel for
that.

I imagine both nift and vercel have diverged a lot since the pr so even if
adding nift/mantis to vercel in the future it's probably easiest to just do
a new pr if/when that ever happens..

Sorry for the semi inappropriate rant, woke up and saw the email and was
reminded about the absurdity of all that 🤦‍♀️

**Pull request closed without merging**